.travis.yml

```diff
@@ -19,6 +19,15 @@ env:
   global:
     - COUCH_URL=http://admin:pass@localhost:5984/medic-test
     - COUCH_NODE_NAME=nonode@nohost
+    - PUBLIC_UPLOAD_URL=https://staging.dev.medicmobile.org
+    - BUILDS_SERVER=_couch/builds_testing
+    - STAGING_SERVER=_couch/builds
+    - EXTERNAL_SERVER=_couch/builds_external
+    - |
+      if [[ -z "$UPLOAD_URL" ]]; then
+        export UPLOAD_URL=$PUBLIC_UPLOAD_URL
```

Comment:
I think it'd be cleaner just to inline `$PUBLIC_UPLOAD_URL` and `$EXTERNAL_SERVER` here because they're not used anywhere else.